csrc/id_model/indexing.cpp

```diff
@@ -457,11 +477,10 @@ Val* TensorIndexer::getLinearIndex(TensorView* tv, const Expr* expr) const {
   // TODO: Contiguous indexing
   Val* index = tv->fusion()->zeroVal();
   for (const auto i : c10::irange(allocation_domains.size())) {
-    // Traverse from innermost to outermost
```

Comment:
I thought this was  the order used in the current indexing, but that is not true.